packages/backend/src/services/ServiceRepository.ts

```diff
@@ -577,6 +580,12 @@ export class ServiceRepository
         );
     }
 
+    public getEmbedService<EmbedServiceImplT>(): EmbedServiceImplT {
+        return this.getService('embedService', () => {
+            throw new Error('EmbedService is not implemented');
```

Comment:
Should we throw the error here or omit the function and let the `getService()` throw `Unable to initialize service '${serviceName}' - no factory or provider.` ?

Reply:
Fair point, I'll let it fall to `getService` to avoid potential confusion later down the road.